app/controllers/admin/videos_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::VideosController < AdminsController
+  def new
+    @video = Video.new
+  end
+
+  def create
+    @video = Video.new(video_params)
+    category = Category.find(params[:video][:categories]) if params[:video][:categories]
+    @video.categories << category
```

Comment:
Will change the M:M to 1:M, fixing the test suits along with the schema change. :)